src/main/java/com/wootecobook/turkey/friend/controller/api/FriendApiController.java

```diff
@@ -0,0 +1,62 @@
+package com.wootecobook.turkey.friend.controller.api;
+
+import com.wootecobook.turkey.commons.resolver.UserSession;
+import com.wootecobook.turkey.friend.service.FriendAskService;
+import com.wootecobook.turkey.friend.service.FriendService;
+import com.wootecobook.turkey.friend.service.dto.FriendAskCreate;
+import com.wootecobook.turkey.friend.service.dto.FriendAskResponse;
+import com.wootecobook.turkey.friend.service.dto.FriendCreate;
+import com.wootecobook.turkey.friend.service.dto.FriendResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/friends")
+public class FriendApiController {
+
+    private FriendAskService friendAskService;
```

Comment:
`final`을 사용하지 않은 이유가 있나요?

```suggestion
    private final FriendAskService friendAskService;
```

Reply:
실수했습니다. 반영했습니다. 감사합니다. 👍